frontend/src/components/story/StoryList.tsx

```diff
@@ -5,17 +5,52 @@ export type StoryListProps = {
   stories: StoryCardProps[] | null;
 };
 
+const NoStoriesFoundCard = () => {
+  // TODO: is this a re-usable component? Deserve its own class?
+  return (
+    <div className="no-stories-found-card">
+      <p>No stories found. Please check access permissions.</p>
+    </div>
+  );
+};
+
 const StoryList = ({ stories }: StoryListProps) => {
-  const storyCards = stories?.map((story: StoryCardProps) => (
-    <StoryCard
-      key={`story-card-key-${story.id}`}
-      id={story.id}
-      title={story.title}
-      description={story.description}
-      youtubeLink={story.youtubeLink}
-      level={story.level}
-    />
-  ));
+  /**
+   * I think this will recalculate everytime we re-render. Is this avoidable? Is having better handled in HomePage?
```

Comment:
I think this operation is arbitrary enough that it isn't a concern. 
Here's a snazzy article if you're thinking about using indexes instead: ["Index as a key is an anti pattern"](https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318)